public/apps/configuration/utils/tenant-utils.tsx

```diff
@@ -78,15 +81,21 @@ export function transformTenantData(
     description: k === globalTenantName ? GLOBAL_USER_DICT.Description : v.description,
     tenantValue: k === globalTenantName ? GLOBAL_USER_DICT.Value : k || GLOBAL_TENANT_SYMBOL,
   }));
-  if (isPrivateEnabled) {
-    // Insert Private Tenant in List
-    tenantList.splice(1, 0, {
-      tenant: PRIVATE_USER_DICT.Label,
-      reserved: true,
-      description: PRIVATE_USER_DICT.Description,
-      tenantValue: PRIVATE_USER_DICT.Value,
-    });
-  }
+  tenantList.splice(1, 0, {
+    tenant: PRIVATE_USER_DICT.Label,
+    reserved: true,
+    description: PRIVATE_USER_DICT.Description,
+    tenantValue: PRIVATE_USER_DICT.Value,
+  });
+  // if (isPrivateEnabled) {
```

Comment:
Looks like this still has not gotten removed yet.

Reply:
@abhivka7 Why adding private tenant to the list anyways? Shouldn't it be added only when private tenant is enabled?

Reply:
@cliu123 With the new UI design changes, we want to always keep private tenant in list. For manage tab, we will have a new column called Status which shows when a tenant is disabled. Removing private tenant from the list cause confusion among users and this new approach of Status column is more descriptive.

Thanks.